src/Pim/Bundle/CatalogBundle/Resources/elasticsearch/index_configuration.yml

```diff
@@ -0,0 +1,45 @@
+mappings:
+    pim_catalog_product:
+        dynamic_templates:
+            -
+                text:
+                    match: '*-text'
+                    match_mapping_type: 'string'
+                    mapping:
+                        fields:
+                            raw:
+                                type: 'keyword'
+                                normalizer: 'text_normalizer'
+                        type: 'text'
+                        analyzer: 'text_analyzer'
+            -
+                varchar:
```

Comment:
Ok, Maybe we can do that in the upcoming Text/TextArea filter (the one we will write the indexing normalizer)